CONTRIBUTING.md.template

```diff
@@ -1,53 +1,31 @@
-We love pull requests from everyone. By participating in this project, you
-agree to abide by the thoughtbot [code of conduct].
+# Contributing
 
-We expect everyone to follow the code of conduct
-anywhere in thoughtbot's project codebases,
-issue trackers, chatrooms, and mailing lists.
+We love pull requests from everyone.
+By participating in this project,
+you agree to abide by the thoughtbot [code of conduct].
 
 [code of conduct]: https://thoughtbot.com/open-source-code-of-conduct
 
-## Dependencies
+We expect everyone to follow the code of conduct
+anywhere in thoughtbot's project codebases,
+issue trackers, chatrooms, and mailing lists.
 
 $(INSTALL_DEPENDENCIES)
 
-## Contributing
-
-1. Fork the repo.
-
-2. Run the tests. We only take pull requests with passing tests, and it's great
-to know that you have a clean slate: $(TEST_RUNNER)
-
-3. Add a test for your change. Only refactoring and documentation changes
-require no new tests. If you are adding functionality or fixing a bug, we need
-a test!
-
-4. Make the test pass.
+Fork the repo.
 
-5. Push to your fork and submit a pull request.
+Make sure the tests pass:
 
+    $(TEST_RUNNER)
 
-At this point you're waiting on us. We like to at least comment on, if not
-accept, pull requests within three business days (and, typically, one business
-day). We may suggest some changes or improvements or alternatives.
+Make your change, with new passing tests. Follow the [style guide][style].
 
-Some things that will increase the chance that your pull request is accepted,
-$(FOR RUBY:
-taken straight from the Ruby on Rails guide:
+[style]: https://github.com/thoughtbot/guides/tree/master/style
 
-* Use Rails idioms and helpers
-* Include tests that fail without your code, and pass with it
-* Update the documentation, the surrounding one, examples elsewhere, guides,
-  whatever is affected by your contribution
+Push to your fork. Write a [good commit message][commit]. Submit a pull request.
 
-Syntax:
+[commit]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
 
-* Two spaces, no tabs.
-* No trailing whitespace. Blank lines should not have any space.
-* Prefer &&/|| over and/or.
-* MyClass.my_method(my_arg) not my_method( my_arg ) or my_method my_arg.
-* a = b and not a=b.
-* Follow the conventions you see used in the source already.
-)
+Wait for feedback. Discuss the feedback. Make any necessary changes.
```

Comment:
I actively dislike reading short, choppy sentences like this (it is the reason I can't read Chuck Palahniuk). They are technically correct sentences, so there's no need to change them for me, but I'd prefer:

> Others will give constructive feedback. This is a time for discussion and improvements, and making the necessary changes will be required before we can merge the contribution.